README.md

```diff
@@ -108,33 +101,16 @@ io.containerd.snapshotter.v1    nydus                    -              ok
 
 ## Quickly Start Container with Lazy Pulling
 
-### Start Container on Node
-
-Containerd can start container with specified snapshotter, so `nerdctl` or `ctr` needs to specify the Nydus snapshotter when start container.
+### Start Container on single Node
 
-A CLI tool [ctr-remote](https://github.com/dragonflyoss/image-service/tree/master/contrib/ctr-remote) is alongside. Use Nydus `ctr-remote` to pull Nydus image or start container based on nydus image.
+Start container using `nydus-snapshotter` using `nerdctl` which has native nydus support.
```

Comment:
Using `nerdctl`, which has native nydus support, to start a container via `nydus-snapshotter`？

Reply:
Fixed